dashboard/app/models/user.rb

```diff
@@ -544,6 +564,13 @@ def update_with_password(params, *options)
     end
   end
 
+  def update_without_password(params, *options)
+    if params[:races]
+      update_column(:races, params[:races].join(','))
+    end
+    super
+  end
```

Comment:
And now I remember... Though the before_save handlers make this redundant as-is, the column is not set independently of the properties key, making it impossible to stop setting the properties key (until something like this is done). Doing so in another PR.